packages/components/table/src/util.ts

```diff
@@ -347,7 +347,9 @@ export function createTablePopper(
   function renderContent(): HTMLDivElement {
     const isLight = tooltipOptions.effect === 'light'
     const content = document.createElement('div')
-    content.className = `${ns}-popper ${isLight ? 'is-light' : 'is-dark'}`
+    content.className = `${ns}-popper ${isLight ? 'is-light' : 'is-dark'} ${
+      tooltipOptions.popperClass || ''
+    }`
```

Comment:
```
[`${ns}-popper`, `${isLight ? 'is-light' : 'is-dark'}`, `${tooltipOptions.popperClass || ''}`].join(' ')
```
Unnecessary spaces can be avoided.

Reply:
I just think the extra spaces won't bring side effects, so it's unnecessary to do something to remove it.

Reply:
I got it.🙈